src/components/VCopyButton.vue

```diff
@@ -55,10 +62,16 @@ export default defineComponent({
       }, 2000)
 
       e.clearSelection()
+
+      /* Set the focus back on the button */
+      setFocusOnButton()
     }
     const onCopyError = (e: Clipboard.Event) => {
       emit('copy-failed')
       e.clearSelection()
+
+      /* Set the focus back on the button as it was*/
```

Comment:
```suggestion
      /* Restore focus on the button */
```

small grammatical suggestion.